packages/tools/replay-tool/src/replayFluidFactories.ts

```diff
@@ -37,37 +31,40 @@ import {
     SharedObjectSequence,
     SparseMatrix,
 } from "@fluid-experimental/sequence-deprecated";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
 import { UnknownChannelFactory } from "./unknownChannel";
-
-async function runtimeRequestHandler(request: IRequest, runtime: IContainerRuntime) {
-    return request.url === "/containerRuntime"
-        ? { mimeType: "fluid/object", status: 200, value: runtime }
-        : create404Response(request);
-}
+import { ReplayToolContainerEntryPoint } from "./helpers";
 
 /** Simple runtime factory that creates a container runtime */
 export class ReplayRuntimeFactory extends RuntimeFactoryHelper {
     constructor(
         private readonly runtimeOptions: IContainerRuntimeOptions,
-        private readonly registries: NamedFluidDataStoreRegistryEntries,
-        private readonly requestHandlers: RuntimeRequestHandler[] = []) {
+        private readonly registries: NamedFluidDataStoreRegistryEntries) {
         super();
     }
 
     public async preInitialize(
         context: IContainerContext,
         existing: boolean,
     ): Promise<ContainerRuntime> {
-        return ContainerRuntime.load(
+        return ContainerRuntime.newLoad(
```

Comment:
The container runtime for the replay tool doesn't seem to use the request pattern? At least not exercised in tests. So moved it to the `newLoad()` method and gave it an entryPoint that just exposes what seems necessary.